vue-components/stories/Message.stories.ts

```diff
@@ -0,0 +1,60 @@
+import Message from '@/components/Message';
+import { Component } from 'vue';
+
+export default {
+	component: Message,
+	// the `/` prefix in the title is needed for "Message" to appear as a folded navigation item, and not a headline
+	title: '/Message/Message',
+};
+
+export function all(): Component {
+	return {
+		components: { Message },
+		template: `
+			<div>
+				<p><Message type="error">An <b>error</b> which means it's potentially stopping whatever the user just did and providing them a clear message on what's wrong. There's also <a href="https://example.com">an explanatory link</a>.</Message></p>
+				<p><Message type="warning">A warning which is not stopping whatever the user just did, but needs to clear up why things might not work as expected.</Message></p>
+				<p><Message type="notice">A neutral message which inform about general higher priority notices to the user. Stay neutral!</Message></p>
+				<p><Message type="success">A success message which means that the user did everything right and should know about it. Yay!</Message></p>
+				<p><Message fullwidth="true" type="success">A success message which means that the user did everything right and should know about it. Yay!</Message></p>
+			</div>
+		`,
+	};
+}
+
+
+export function controllableMessage(args): Component {
```

Comment:
Currently, due to a bug I'm trying to resolve [T265039](https://phabricator.wikimedia.org/T265039), type checking isn't working. However, I think we need to add a type here.

```suggestion
export function controllableMessage( args: object ): Component {
```

Reply:
But shouldn't that have failed at _some point_ then? Even the job that did build the storybook for Chromatic is green?

But I can update it with its actual type.

Reply:
like I said, it currently doesn't fail due to a bug I am working on.

Reply:
Ah sorry, I've misread that ticket as saying that it fails only in the build process.

Reply:
I think that should now be fixed :)